### DISCUSSION TITLE
Set browser window title

### DISCUSSION THREAD

**Discussion by monder007** [filamentphp/filament#100] [Help] [answered]
Hi

Is it possible to set the browser window title per resource?

In the `base.blade.php` it looks like i should be able to set some `$title` variable...

**Comment by ryangjchandler** [comment, marked as answer]
Hey @monder007, thanks for opening the discussion. Have you got any thoughts about a preferred API for setting the page / window title?

I can imagine a scenario where you'd want to change the title based on the current resource, as well as a standard page title on list / create screens.

**Reply by ryangjchandler** [reply]
@monder007 I've opened up a pull request with a potential solution - will wait for others to review first, but would this solve your issue? https://github.com/laravel-filament/filament/pull/102

**Reply by monder007** [reply]
Hi @ryangjchandler 

Looks perfect. So it should set it to a couple of defaults and then to fully override it, we should be able to set 

`public static $pageTitle`

on the resource to customise it.

**Reply by ryangjchandler** [reply]
Yeah that's right. Or if you needed something more dynamic that can't be assigned to a property, you can override the `getPageTitle` method.

**Comment by usamamuneerchaudhary** [comment]
This is older but came on google search. Leaving a comment for v3. it can be done like;

```
public function getTitle(): string
    {
        return 'My Page title';
    }
